### PR TITLE
Help me Melvin Conway -- You're my only hope!

### PR DESCRIPTION
**Pull request by johnsyweb**
Context
--------

Following a conversation with @jayvee this morning and [my strong, weakly-held opinion](https://envato.slack.com/archives/marketplacedev/p1470292242001247) that we should archive [#marketplacedev](https://envato.slack.com/archives/marketplacedev).

Change
--------

- Move Seal outputs to where they belong

Complementary and complimentary image
--------------------------------------------

conways-law-continious-delivery-5-638.

Conversation
---------------

Discuss, in https://envato.slack.com/archives/customer-engineering/p1475800984000032

### CODE REVIEW

**Comment by johnsyweb** [comment]
Controversy
-------------

Months after the reorganisation, Customer, Content and Platform Finance folks are still making changes to the Customer Monolith. Our organisational communications encourage this behaviour to continue. Let's make a clean break!

**Comment by jayvee** [comment]
👍

**Comment by johnsyweb** [comment]
Anyone else? Cc: @envato/marketplace

**Comment by vonconrad** [comment]
:+1:

**Comment by asellitt** [comment]
Im confused. we're talking about archiving the slack channel (this is a public repo btw)? Or are we suggesting we rename the slack channel to #customer-engineering? Or are we just removing the seal spam?

:|

**Comment by johnsyweb** [comment]
>Im confused. 

Then I am sorry. I wasn't clear enough in the context above.

> we're talking about archiving the slack channel (this is a public repo btw)?

We are. It should be history by now (IMO).

> Or are we suggesting we rename the slack channel to #customer-engineering?

This already exists. You should join!

> Or are we just removing the seal spam?

We're moving the Seal information as a first step.

:-)

**Comment by asellitt** [comment]
ok. My thoughts here (in no particular or filtered order):

- as much as we seem to love the idea of autonomous teams, we're still all shackled to the monolith
- the work of one team is MOST CERTAINLY going to impact on another (because monolith)
- this is a public channel, not comfortable with being 100% candid
- as long as i can be a good and sensible dev and ask my fellow "prisoners" for their review/approval somehow, i dont care what the name of the channel is.
- the seal is very disruptive for early morning conversations
- i better join #customer-engineering
- im likely to remain grumpy at the seal in #customer-engineering

TL;DR: sounds like nothing is changing except for where we're pointing the sealhose. *shrugs* no solid opinions for or against, so 👍  from me.

**Comment by petervandoros** [comment]
👍

**Comment by amanda08** [comment]
Are we revoking non customer-team access to the marketplace repo as well?
The way I see it it's not possible to make a clean break until we have new systems to replace the existing monolith.

**Comment by AeroCross** [comment]
I think the problem is the Seal and not the Slack channel.

I like to be able to post a question, or a comment, or being able to get reviews from **anyone** in the **Market Team**, not just my team (or wider group — we're still just like 9 devs).

We may be working in different _areas_ of the Market, but we're still the Market engineering team.

**Comment by nocache** [comment]
After archiving `#marketplacedev`, where should conversations that are relevant to all people working on the market places be conducted?

**Comment by twe4ked** [comment]
:+1: to removing Seal
:-1: to archiving #marketplacedev until we're no longer required to work on the Market monolith

**Comment by AeroCross** [comment]
Exactly, @nocache.

**Comment by patrobinson** [comment]
> as much as we seem to love the idea of autonomous teams, we're still all shackled to the monolith

This captures my main concern with this approach

> organizations which design systems ... are constrained to produce designs which are copies of the communication structures of these organizations

I agree with this sentiment and hence the marketplace is a monolith, it represented the structure of the organisation _at that point in time_. Changing the organisational structure does not automatically make the system reflect that.

> my strong, weakly-held opinion that we should archive #marketplacedev.

There is still considerable work to de-couple the monolith, until that happens then the requirement to collaborate on most changes to the monolith between groups is required. Within launch standup they have a concept of a "project circle", that is a group of teams collaborating on a project. Since this is a monolith, it's inevitable that almost every change requires that collaboration. A slack channel is a pseudo-project circle (and a very efficient one in my opinion).

So I'm 👍  to removing seal, for other reasons, and 👎  to archiving #marketplacedev

**Comment by gstamp** [comment]
> Changing the organisational structure does not automatically make the system reflect that.

Correct. We've got a long way to go there. 

Personally I'm fine with archiving #marketplacedev but honestly I don't think it will make one bit of difference to our day to day work. One thing that does impact me is that there are so many channels that I only have a vague idea of what they're for. Spreading my attention across so many channels is problematic. Figuring out which is the right one for any given query can be harder than it should be. A lot of them don't even have reasonable topics set.

PS. I'm surprised that people don't like the seal since it only posts once per day.

**Comment by asellitt** [comment]
i dont like seal because of **when** it posts. I actively avoid chats in mpdev before 9am because of the scrollspam. If it was splatted earlier (12am that morning, or any other time out of hours) then it wouldnt interrupt and would be fine.

**Comment by asellitt** [comment]
interestingly this PR proposes a `move` not a `remove`. Such controversy! should we consider killing it then?

**Comment by macosgrove** [comment]
I'm also having difficulty figuring out which Slack channel to post queries in. As long as this can be made clear, I don't mind what the names are, or how many of them there are.

**Comment by johnsyweb** [comment]
I think I have enough 👍 's to merge this.

Discussions on Organisational Structures / Communication Channels can happen in a private channel.

**Comment by gusgollings** [comment]
👍 Thanks @johnsyweb - this all makes good sense (no more Market team, we want to feel the dissonance of crossing team boundaries, need to discuss this all within the group).
